src/amberscript/parser.cc

```diff
@@ -1718,6 +1718,99 @@ Result Parser::ParseRun() {
     return ValidateEndOfStatement("RUN command");
   }
 
+  if (token->AsString() == "DRAW_GRID") {
+    if (!pipeline->IsGraphics())
+      return Result("RUN command requires graphics pipeline");
+
+    if (pipeline->GetVertexBuffers().size() > 1) {
```

Comment:
The comment above says the index and vertex data are ignored. Why do we allow one vertex buffer to be attached if it will be ignored?

Reply:
Copied off from draw_rect. Fixed here, but not in draw_rect.